client/assets/javascripts/modules/googleAnalyticsEvents.js

```diff
@@ -0,0 +1,29 @@
+class GoogleAnalyticsEvents {
+  static init () {
+    const elements = document.querySelectorAll('button[data-attribute="ga-event"]')
+
+    elements.forEach(element => {
```

Comment:
Better to use event delegation for this (https://davidwalsh.name/event-delegate). Listen for click at the document level and only respond if it has a matching ga-event attribute.

Reply:
Nice, that's a much better approach 👍